.github/workflows/lint-ios.yml

```diff
@@ -5,6 +5,9 @@ on:
 #    branches: [ develop, main ]
 jobs:
   test-iOS:
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
Why do we need `write` permission for `pull-requests`? Seems like we shouldn't need it at all, or at most `read`?